### PR TITLE
[ta] 03 - Defining test cases and priorities

### LINKED ISSUE #10227: [ta] 03 - Defining test cases
Published date:  Aug 9

### PR DESCRIPTION
**Pull request by sofiayem**
@jecfish, is the hero image ok? Feel free to suggest a better one.
Fixes #10227

### CODE REVIEW

File: src/site/content/en/blog/ta-test-cases/index.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+title: Defining test cases and priorities
+date: 2023-08-09
+scheduled: true
+subhead: >
+  Determine what to test, define your test cases, and prioritize.
+description: >
+  Determine what to test, define your test cases, and prioritize.
+authors:
+  - ramona
+hero: image/NJdAV9UgKuN8AhoaPBquL7giZQo1/eVRJ3GRUx8h4MFaBATut.jpg
```
Comment (jecfish) [inline src/site/content/en/blog/ta-test-cases/index.md:12]:
```suggestion
hero: image/dPDCek3EhZgLQPGtEG3y0fTn4v82/reTqGo078zXM3UQ9wgHf.jpg
```

**Review by jecfish — APPROVED** [review]

File: src/site/content/en/blog/ta-test-cases/index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
+---
+layout: post
+title: Defining test cases and priorities
+date: 2023-08-09
+scheduled: true
```
Comment (jecfish) [inline src/site/content/en/blog/ta-test-cases/index.md:5]:
```suggestion
```

**Comment by jecfish** [comment]
@sofiayem I would like to merge this today but I couldn't because it needs another tech writer review. How can we proceed?

**Comment by jecfish** [comment]
thanks Rachel!

**Comment by rachelandrew** [comment]
@jecfish if there are deadlines for these can you let me know what they are? Otherwise they wait in my queue for review.

**Comment by jecfish** [comment]
Okay, I usually mentioned the deadline in the issue #10227. I think what we can do is Sofia can put in the deadline when she create this PR. 👍

**Comment by rachelandrew** [comment]
@jecfish Can we make sure we have edited content a week before those expected dates? I can see in your doc publish dates, but the content should be ready to go well in advance of that. These have required a lot of work on our side so I'd like to see the approved content in docs format at least a week before you want to publish.

**Pull request merged**